test/test.ts

```diff
@@ -138,6 +139,19 @@ describe('lift', () => {
       expect(updated).toNotBe(arr)
     })
 
+    it('won\'t replace an item if the given index is out of bounds', () => {
```

Comment:
In general I write `it('will not replace...` so I don't have to deal with escaped quotes.

No need to change it though. Merging as is.